src/Presentation/Nop.Web.Framework/Migrations/UpgradeTo450/LocalizationMigration.cs

```diff
@@ -169,7 +169,7 @@ public override void Up()
                 ["Account.Login.NewCustomerText"] = "By creating an account on our website, you will be able to shop faster, be up to date on an order's status, and keep track of the orders you have previously made.",
 
                 ["Permission.SalesSummaryReport"] = "Admin area. Access sales summary report",
-                ["Permission.ManageAppSettings"] = "Admin area. Manage App Setting",
+                ["Permission.ManageAppSettings"] = "Admin area. Manage App Settings",
```

Comment:
Hi @curllog. These changes do not make sense, many users have already completed this migration, and corrections need to be made to the migration for 4.70

Reply:
@skoshelev  thanks for your response so we don't need change at the moment? do i need to close this pr?

Reply:
@skoshelev i changed this change in 4.70 migration can you check changes please?